package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mov-ai/mov-fe-app-ide-ce",
```

Comment:
Can you please change the package name in package.json and package-lock.json?
name should be `"@mov-ai/mov-fe-lib-ide"`